docs/configuration.md

```diff
@@ -300,6 +300,29 @@ node-local load balancing.
 | `apiServerBindPort`          | Port number on which to bind the Envoy load balancer for the Kubernetes API server to on a worker's loopback interface.  Default: `7443`. |
 | `konnectivityServerBindPort` | Port number on which to bind the Envoy load balancer for the konnectivity server to on a worker's loopback interface. Default: `7132`.    |
 
+##### `spec.network.controlPlaneLoadBalancing`
+
+Configuration options related to k0s's [control plane load balancing] feature
+
+| Element         | Description                                                                               |
+| --------------- | ----------------------------------------------------------------------------------------- |
+| `vrrpInstances` | Configuration options related to the VRRPInstances. This is used to configure virtual IPs |
+
+[control plane load balancing]: cplb.md
+
+##### `spec.network.controlPlaneLoadBalancing.VRRPInstances`
+
+Configuration options required for using VRRP to configure VIPs in control plane load balancing.
+
+| Element           | Description                                                                                        |
+| ----------------- | -------------------------------------------------------------------------------------------------- |
+| `name`            | The name of the VRRP instance. If omitted it generates a predictive name shared accross all nodes. |
+| `virtualIPs`      | A list of the CIDRs handled by the VRRP instance.                                                  |
+| `interface`       | The interface used by each VRRPInstance. Default: `eth0`.                                          |
```

Comment:
Couldn't we somehow try to autodetect the interface?

Reply:
That makes sense, I'll use the interface used for the default gateway